pom.xml

```diff
@@ -389,6 +389,21 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
[Question] I found "The Surefire Plugin is used during the test phase of the build lifecycle to execute the unit tests of an application. It generates reports in two different file formats." Just wondering how does your change require this plugin?

Reply:
In fact, I do not encounter the message `The Surefire Plugin is used during the test phase of the build lifecycle to execute the unit tests of an application. It generates reports in two different file formats.` you mentioned. But the purpose of this part of the `maven-surefire-plugin` plugin is to skip running test cases when building and packing this jar library. So, it should be equivalent to `-DskipTests` option to maven command. Since I knew using plugin to skip tests before knew the `-DskipTests` and the compiled custom `ews-java-api` jar has included it. So, to consistency, I keep it here.

Reply:
The similar thing to the part of the plugin to create checksum, it should be equivalent to `-DcreateChecksum=true` option to maven command.

Reply:
Or, I could remove these two part of plugins from the pom file and then re-compile with both `-DskipTests -DcreateChecksum=true` options to maven command. Does it solve your concern about the "The Surefire Plugin is used during the test phase of the build lifecycle to execute the unit tests of an application. It generates reports in two different file formats."?

Reply:
Thanks Jiaju. I wanted to know what is the Surefire plugin used for, so I searched online and found the sentence in the introduction of  Surefire Plugin https://maven.apache.org/surefire-archives/surefire-3.0.0-M2/maven-surefire-plugin/index.html#:~:text=The%20Surefire%20Plugin%20is%20used,txt). I think it's ok to keep the plugin if you want to skip tests by default. Same for install plugin to create checksum by default.

Reply:
Since I notice an test failure (`testOlsonTimeZoneConversion(microsoft.exchange.webservices.data.property.complex.OlsonTimeZoneTest)  Time elapsed: 0.051 sec  <<< FAILURE!`) when building the jar library. And I think it's a minor issue and it should not affects the ordinary/normal use cases. So, I have to skip all the test. Otherwise, I need to provide a further fix to solve it.